File: 17487.py
right_hands = ['U', 'u', 'I', 'i', 'O', 'o', 'P', 'p', 'H', 'h', 'J', 'j', 'K', 'k', 'L', 'l', 'N', 'n', 'M', 'm']

S = input()
left = right = shift = space = 0
for i in range(len(S)):
    if S[i] == ' ': space += 1
    else:
        if S[i].isupper(): shift += 1
        if S[i] in right_hands:
            right += 1
        else: left += 1

for i in range(shift):
    if left <= right: left += 1
    else: right += 1
for i in range(space):
    if left <= right: left += 1
    else: right += 1

print(left, right)

File: 17487.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
	string S; getline(cin, S);

	int left, right, shift, space;
	left = right = shift = space = 0;
	for (int i = 0; i < S.length(); i++) {
		if (S[i] == ' ') space++;
		else {
			if (isupper(S[i])) shift++;
			if (S[i] == 'U' || S[i] == 'u' || S[i] == 'I' || S[i] == 'i'
				|| S[i] == 'O' || S[i] == 'o' || S[i] == 'P' || S[i] == 'p'
				|| S[i] == 'H' || S[i] == 'h' || S[i] == 'J' || S[i] == 'j'
				|| S[i] == 'K' || S[i] == 'k' || S[i] == 'L' || S[i] == 'l'
				|| S[i] == 'N' || S[i] == 'n' || S[i] == 'M' || S[i] == 'm')
				right++;
			else left++;
		}
	}

	for (int i = 0; i < shift; i++) {
		if (left <= right) left++;
		else right++;
	}
	for (int i = 0; i < space; i++) {
		if (left <= right) left++;
		else right++;
	}

	cout << left << ' ' << right << endl;
}
